gravitee-am-service/src/main/java/io/gravitee/am/service/impl/DomainServiceImpl.java

```diff
@@ -615,6 +616,13 @@ private Completable validateDomain(Domain domain) {
             }
         }
 
+        if (domain.getWebAuthnSettings() != null) {
+            final String origin = domain.getWebAuthnSettings().getOrigin();
+            if (origin == null || origin.isBlank()) {
```

Comment:
We should also test that it is a valid URL (starting by `http` or `https`)

Reply:
scheme checked in back end. From front end it looks line